### PR TITLE
Fixes IP address formatting #2109

### PR DESCRIPTION
**Pull request by ujjwal96**
Since other PR was closed by the author
Should I make changes in flowdetailview as well?

### CODE REVIEW

**Comment by mhils** [comment]
Yes, please. Thanks for picking this up! :smiley:

**Comment by ujjwal96** [comment]
@mhils This is ready for review :slightly_smiling_face:

File: mitmproxy/tools/console/flowdetailview.py [hunk 1]
```diff
@@ -30,8 +30,8 @@ def flowdetails(state, flow: http.HTTPFlow):
     if sc is not None:
         text.append(urwid.Text([("head", "Server Connection:")]))
         parts = [
-            ["Address", "{}:{}".format(sc.address[0], sc.address[1])],
-            ["Resolved Address", "{}:{}".format(sc.ip_address[0], sc.ip_address[1])],
+            ["Address", "{}".format(human.format_address(sc.address))],
+            ["Resolved Address", "{}".format(human.format_address(sc.ip_address))],
```
Comment (Kriechi) [inline mitmproxy/tools/console/flowdetailview.py]: Using `"{}".format(...)` is a bit beside the point... 😄

Comment (ujjwal96) [inline mitmproxy/tools/console/flowdetailview.py]: Heh, right 😅 
Will remember to change it in my next PR 🙂

**Pull request merged**